src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search {
+  display: block;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.bigSearch__input {
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  margin-top: 20px;
+  padding-left: 62px;
+  background: url(images/Search.svg) no-repeat 26px 25px;
+}
+
+.bigSearch__input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigSearch__input:focus {
+  background: url(images/Search.svg) no-repeat 26px 25px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
you can add this rules to .search:focus selector and
![image](https://user-images.githubusercontent.com/54936044/232627503-d0019b1a-3e6a-4b37-9bad-22be2eed17d0.png)
and don't repeat them.

Reply:
There is difference size ...

Reply:
I put it wrong, I meant that you can focus the gradient in the general class, and set the image through the background-image rule. As a result, you won't need two of these selectors